### PR TITLE
add testcase for login and register

### PR DESCRIPTION
**Pull request by plumrx**

### CODE REVIEW

File: cypress/integration/register.spec.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    cy.visit('/')
+  })
+
+  it('ligin in home page', () => {
+    // click logup button in home page
+    const usrname = 'plumrx'
+    const email = 'plumrx@qq.com'
+    const password = '12345678'
+    cy.register(usrname, email, password)
```
Comment (mutoe) [inline cypress/integration/register.spec.js:18]: You can abstract the arguments like your `cy.login` command 😄

File: cypress/integration/login.spec.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    cy.route2('GET', /articles\//, { fixture: 'article.json' }).as('getArticle')
+    cy.route2('GET', /tags/, { fixture: 'tags.json' }).as('getTags')
+
+    cy.visit('/')
+  })
+
+  it('login in home page', () => {
+    // const usrname = 'plumrx1'
+    // const email = 'plumrx1@qq.com'
+    // const password = '12345678'
```
Comment (mutoe) [inline cypress/integration/login.spec.js:15]: You can remove those lines

File: cypress/support/commands.js [hunk 3]
```diff
@@ -23,3 +23,39 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', (user) => {
```
Comment (mutoe) [inline cypress/support/commands.js:27]: Very good custom command!

**Review by mutoe — CHANGES_REQUESTED** [review]

File: cypress/integration/register.spec.js (hunk 1 above)
Comment (plumrx) [reply cypress/integration/register.spec.js:18]: Fine, I will fix it tomorrow

**Review by mutoe — APPROVED** [review]

**Comment by levchak0910** [comment]
@plumrx, further, please make sure that you are up-to-date with origin/master, before creating PR

**Comment by plumrx** [comment]
> @plumrx, further, please make sure that you are up-to-date with origin/master, before creating PR
> origin = mutoe/vue3-realworld-example-app

@levchak0910 Sorry, I am a beginner, I will pay attention to the problem you mentioned next time.

**Comment by levchak0910** [comment]
@plumrx, it's ok, just a hint

**Pull request merged**